stingray/io.py

```diff
@@ -653,3 +654,31 @@ def get_btis(gtis, start_time=None, stop_time=None):
 def gti_len(gti):
     """Return the total good time from a list of GTIs."""
     return np.sum([g[1] - g[0] for g in gti])
+
+
+def save_object(object, filename):
+    """
+    Pickle a class instance.
+
+    Parameters
+    ----------
+    object: a class instance
+    filename: str (name of the file to be created)
+
+    """
+
+    filename = "stingray/data/" + filename
```

Comment:
I don't feel very comfortable with this. If a user installed the software using pip or conda and does not has the source code of the repository, then they will have to create directories named `stingray` and `data` for reading and writing purpose. This also sets a limitation that they can only use the data file from that location. Please correct me if I am wrong.

Alternative solutions : 

1. If we simply use `filename` then that would mean the user can read/write from the current location, wherever they are.

2. (Not recommended) Use a static `data` directory like the Home or the installation directory.